src/models/link.model.ts

```diff
@@ -0,0 +1,27 @@
+export enum LinkStatus {
+  inactive = 0,
```

Comment:
Need not specify the value, it will take zero by default
``` typescript
var Sample;
(function (Sample) {
    Sample[Sample["valOne"] = 0] = "valOne";
    Sample[Sample["valTwo"] = 1] = "valTwo";
    Sample[Sample["valThree"] = 2] = "valThree";
    Sample[Sample["valFour"] = 3] = "valFour";
})(Sample || (Sample = {}));
```
this is how it converts to.